Makefile

```diff
@@ -4,6 +4,20 @@ IMAGE_REGISTRY ?= quay.io
 IMAGE_REPO ?= nmstate
 NAMESPACE ?= nmstate
 
+NMSTATE_CURRENT_COPR_REPO=nmstate-0.3
+NM_CURRENT_COPR_REPO=NetworkManager-1.26
+
+NMSTATE_FUTURE_COPR_REPO=nmstate
+NM_FUTURE_COPR_REPO=NetworkManager-1.26
+
+ifeq ($(NMSTATE_PIN), future)
+	NMSTATE_COPR_REPO=$(NMSTATE_FUTURE_COPR_REPO)
```

Comment:
We could indent other ifeq/else blocks in this Makefile as well, but that doesn't concern this PR.